ui/design-system-components/src/components/Profile/Links/index.tsx

```diff
@@ -17,24 +23,39 @@ export type LinksProps = {
 };
 
 const Links: React.FC<LinksProps> = ({ title, links }) => {
+  const getIconFromLink = (href: string) => {
+    const host = new URL(href).host;
+
+    if ('github.com'.includes(host)) return { icon: <Github />, solid: false };
+    if ('twitter.com'.includes(host)) return { icon: <Twitter />, solid: true };
+    if ('discordapp.com'.includes(host)) return { icon: <Discord />, solid: true };
+    if ('telegram.me'.includes(host) || 't.me'.includes(host))
+      return { icon: <Telegram />, solid: true };
+    return { icon: <LinkIcon />, solid: false };
+  };
```

Comment:
This function expression will be re-created needlessly every time the component re-renders. Either we need to put it inside a useCallback or create a function outside of the component.